elastalert/elastalert.py

```diff
@@ -530,7 +530,8 @@ def get_hits_terms(self, rule, starttime, endtime, index, key, qk=None, size=Non
             buckets = res['aggregations']['filtered']['counts']['buckets']
         else:
             buckets = res['aggregations']['counts']['buckets']
-        self.thread_data.num_hits += len(buckets)
+        if getattr(self.thread_data,"num_hits",None):
```

Comment:
This fixes a bug I met when having `top_count_keys` in a frequency rule with `aggregation`.